extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -129,15 +152,36 @@
                 "        if (headers):\n",
                 "            for header in headers:\n",
                 "                title = header[10:].rstrip()\n",
-                "                contentFolders.append(title.lower())\n",
-                "                filtered = list(filter(lambda x: (\"%s%s%s\" % (os.path.sep, title.lower(), os.path.sep)) in x, urls))\n",
+                "                filtered = list(filter(lambda x: (\"%s%s%s\" % (os.path.sep, title.lower(), os.path.sep)) in x.lower(), urls))\n",
                 "                index = urls.index(filtered[len(filtered)-1])\n",
+                "                markdownFilePath = getMarkdownFile(filtered[len(filtered)-1])\n",
+                "                markdownExists = path.exists(markdownFilePath)\n",
```

Comment:
In general please add more comments explaining what each step is doing. It's very unclear what all this logic is for which makes it a lot harder to determine if it's doing the right thing.

Reply:
I did add some comments around, but I still need to do better. Most of my time is gone fiddling around python code since am new it. But I'll make sure add some more comments in a separate PR.